Jet_fitting_3/doc/Jet_fitting_3/Jet_fitting_3.txt

```diff
@@ -224,7 +224,7 @@ In addition, the class `Monge_via_jet_fitting` stores
 This concept provides the types for the input sample points, together
 with \f$ 3d\f$ vectors and a number type. It is used as template for the
 class `Monge_via_jet_fitting<DataKernel, LocalKernel, SvdTraits>`. Typically, one can use
-`Cartesian<double>`.
+`Cartesian< double >`.
```

Comment:
Why does that matter? If it does, maybe we should explain why in the guidelines.

Reply:
It was understood as a html tag by doxygen.